docs/source/10min.rst

```diff
@@ -0,0 +1,398 @@
+10 Minutes to cuDF
+=======================
+
+Modeled after 10 Minutes to Pandas, this is a short introduction to cuDF, geared mainly for new users.
+
+.. ipython:: python
+   :suppress:
+
+   import os
+   import numpy as np
+   import pandas as pd
+   import cudf
+   np.random.seed(12)
+
+   #### Portions of this were borrowed from the
+   #### cuDF cheatsheet, existing cuDF documentation,
+   #### and 10 Minutes to Pandas.
+   #### Created November, 2018.
+
+
+Object Creation
+---------------
+
+Creating a `Series`.
+
+.. ipython:: python
+
+    s = cudf.Series([1,2,3,None,4])
+    print(s)
```

Comment:
Can you show the output of each print / section?

Reply:
The sphinx extension automatically generates the output when you make the rst file. We could include it as suppressed output/comments explicitly in the file if you think people will want to look at the raw .rst source file, but I think that would create clutter. Do you think auto-generating the output is enough?

Reply:
For reference: https://github.com/pandas-dev/pandas/blob/master/doc/source/10min.rst

Reply:
Ah, didn't know that. This is fine. :)